Lavanya/HTMLassignment2.html

```diff
@@ -0,0 +1,51 @@
+<html>
+
+<head>
+	<style>
+		table,
+		th,
+		td {
+			border: 1px solid black;
+		}
+	</style>
+</head>

```

Comment:
Add necessary tags within the head tag. Refer this: [HTML Head - w3schools.com](https://www.w3schools.com/html/html_head.asp)